Doc/library/sysconfig.rst

```diff
@@ -185,7 +185,7 @@ Other functions
 
    Windows will return one of:
 
-   - win-amd64 (64bit Windows on AMD64 (aka x86_64, Intel64, EM64T, etc)
+   - win-amd64 (64bit Windows on AMD64, aka x86_64, Intel64, EM64T)
```

Comment:
Why was "etc" removed? If remove it, should not "and" be added?

Reply:
Because I think "etc" should only be used when a sequence can be deducted (e.g.: 'a', 'b', 'c', etc.).

I agree with the "and".